pkgs/applications/networking/remote/vmware-horizon-client/default.nix

```diff
@@ -37,7 +37,7 @@
 , zlib
 }:
 let
-  version = "2006";
+  version = "2012";
```

Comment:
Why is the version different from the one in the URL? Maybe this should be unified and an update script added?

Reply:
The URL has both 8.1.0 and 2012. All their releases have two version numbers now. The 2006/2012 versioning is newer, so it seems like they are switching away from the 8.x and toward the [YYMM] format. Also when you go to download it the version dropdown has the 2012 version.

Reply:
I'm not sure how the update script will work since at least part of the URL is always random, Maybe some website scraping can do it, but haven't looked into it yet.

Reply:
Converted to a Draft while I look into an update script. It seems less impossible than I previously thought.